src/cmd-buildextend-installer

```diff
@@ -180,25 +201,43 @@ def generate_iso():
         os.chmod(os.path.join(tmpisoimages, file), 0o644)
 
     # Generate initramfs stamp file which says whether it's a live or legacy
-    # initramfs.
+    # initramfs.  Store a cookie unique to this build.
+    build_cookie = base64.urlsafe_b64encode(os.urandom(33)).decode() + '\n'
```

Comment:
The `cookie` just needs to be unique, right? Since our build versions are already unique would it be worth us just making the cookie the version number, this would make it easy for us to tell the user: "the error you are getting is because you are using a initramfs from `32.20200629.3.0` but a rootfs from `32.20200615.3.0`.

Reply:
The unique cookie catches the case where `buildextend-live --force` is rerun and we're using an initramfs/rootfs from two different runs.  But that's a pretty obscure case, and you're right that we could improve the UX by embedding the version.

Reply:
+1 - i'm cool either way, just another option.

Reply:
Done.